packages/bitcore-wallet-client/src/lib/credentials.ts

```diff
@@ -176,6 +176,8 @@ export class Credentials {
         coin = '0';
       } else if (this.coin == 'eth') {
         coin = '60';
+      } else if (this.coin == 'xrp') {
+        coin = '144';
```

Comment:
we should have a method for this in CWC, if not we should add that

Reply:
https://github.com/bitpay/bitcore/pull/2640